lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1879,17 +1877,24 @@ def plot_wireframe(self, X, Y, Z, *args, **kwargs):
     def plot_trisurf(self, *args, color=None, norm=None, vmin=None, vmax=None,
                      lightsource=None, **kwargs):
         """
-        ============= ================================================
-        Argument      Description
-        ============= ================================================
-        *X*, *Y*, *Z* Data values as 1D arrays
-        *color*       Color of the surface patches
-        *cmap*        A colormap for the surface patches.
-        *norm*        An instance of Normalize to map values to colors
-        *vmin*        Minimum value to map
-        *vmax*        Maximum value to map
-        *shade*       Whether to shade the facecolors
-        ============= ================================================
+
+        Parameters
+        ----------
+        X, Y, Z : array-like
+            Data values as 1D arrays
+        color
+            Color of the surface patches
+        cmap
+            A colormap for the surface patches.
+        norm : Normalize
+            An instance of Normalize to map values to colors.
+        vmin, vmax : scalar, optional, default: None
+            Minimum and maximum value to map.
+        shade : bool
+            Whether to shade the facecolors. Defaults to True. Shading is
+            always disabled when `cmap` is specified.
+        lightsource : `~matplotlib.colors.LightSource`
+            The lightsource to use when `shade` is True.
 
         The (optional) triangulation can be specified in one of two ways;
```

Comment:
There can be no text block in a parameters section in numpydoc style. This would be interpreted as further parameters, which [doesn't look nice](https://12589-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/mpl_toolkits.mplot3d.axes3d.Axes3D.html#mpl_toolkits.mplot3d.axes3d.Axes3D.plot_trisurf)

Please move this block before the parameters section.